frontend/src/App.vue

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 -->
 
 <template>
-  <router-view></router-view>
+  <router-view :key="$route.path"></router-view>
```

Comment:
why?

Reply:
switching between cluster and control plane terminal does not work if the key is not set

Reply:
fixed differently. Done